### ISSUE TITLE
Feature to upload existing post to story

### ISSUE THREAD

**Issue by pervaiz425** [subzeroid/instagrapi#1677]
Hi Coders,

This library does not have the functionality to upload existing post to story. I think this is the major lack in this library. Let's make it more robust. I have built a general template for this. We could build a new method for this. Looking forward to hear your responses.

Best,
Muhammad

**Comment by gauravsingh06** [comment]
yes, I was searching for this, It is very important feature. It's the only feature I think I am missing in this lib

**Comment by subzeroid** [comment]
Keeping this open as a backlog feature request. This is the native Instagram "share an existing post to Story" flow, which is different from uploading a prepared photo/video as a Story. We’ll look into whether the mobile flow can be reproduced cleanly and exposed as a separate helper.

**Comment by subzeroid** [comment]
Implemented in #2572: adds `Client.media_share_to_story(...)` as a convenience helper for sharing an existing feed post to Story via the feed-media sticker story flow. It generates a temporary Story background when one is not provided, documents the helper, and includes regression plus targeted live coverage.

**Comment by subzeroid** [comment]
Released in [2.7.20](https://github.com/subzeroid/instagrapi/releases/tag/2.7.20). The package is available on PyPI as .

**Comment by subzeroid** [comment]
Released in [2.7.20](https://github.com/subzeroid/instagrapi/releases/tag/2.7.20). The package is available on PyPI as `instagrapi==2.7.20`.
